### PR TITLE
Dev #T1062: When user clicks the reset button the screen freezes while should reset to new default

### PR DESCRIPTION
**Pull request by gabrieljenik**
Added "reset" buttons for the specific screens on FullPagebar.

### CODE REVIEW

File: application/controllers/admin/SurveymenuController.php [hunk 1]
```diff
@@ -391,7 +391,8 @@ public function view()
                 'buttons' => [
                     'addMenu' => true,
                     'addMenuEntry' => true,
-                    'reset' => Permission::model()->hasGlobalPermission('superadmin', 'read'),
+                    'resetMenus' => Permission::model()->hasGlobalPermission('superadmin', 'read'),
+                    'resetMenuEntries' => true, // TODO: Is this OK? It wasn't checking any permission on SurveymenuEntryController
```
Comment (Shnoulle) [inline application/controllers/admin/SurveymenuController.php:395]: Unsure ;  Permission::model()->hasGlobalPermission('superadmin', 'read'), ?

But tottaly unsure here … there are user menu entries ?

File: application/views/admin/super/fullpagebar_view.php [hunk 2]
```diff
@@ -99,6 +99,29 @@ class='btn btn-default'
                 </a>
             <?php endif; ?>
 
+            <!-- Reset Survey Menus -->
+            <?php if (isset($fullpagebar['menus']['buttons']['resetMenus']) && $fullpagebar['menus']['buttons']['resetMenus']) :?>
```
Comment (Shnoulle) [inline application/views/admin/super/fullpagebar_view.php:103]: I always prefer : `if (!empty($fullpagebar['menus']['buttons']['resetMenus'])`

shorter, and allow 0/""/null/false/not set for false

Like other dev want :)

**Review by Shnoulle — COMMENTED** [review]
Code is OK

**Review by Shnoulle — CHANGES_REQUESTED** [review]
Code OK, some remark not linked to code + conflict

**Comment by gabrieljenik** [comment]
I think this can be closed.
Seems this has already been solved differently on master.
Zoho task is marked as done.

**Pull request closed without merging**